src/core/dto/social-feed.dto.ts

```diff
@@ -55,3 +55,24 @@ export class UpdatePostInput {
   @Field(() => Date, { nullable: true })
   updatedAt?: Date;
 }
+
+@InputType()
+export class CreateCommentInput {
+  @Field(() => String, { nullable: true })
+  content?: string;
+
+  @Field(() => Date, { nullable: true })
+  createdAt?: Date;
+
+  @Field(() => Date, { nullable: true })
+  updatedAt?: Date;
+
+  @Field(() => Int, { nullable: true })
+  postId?: number;
+}
+
+@InputType()
+export class UpdateCommentInput {
+  @Field(() => String, { nullable: true })
+  content?: string;
+}
```

Comment:
I would remove this one, the [design handoff figma](https://www.figma.com/file/tXuNcaYzPZjKGeFR49bldP/SPI-UXD-to-Dev-Handoff?type=design&node-id=7811-7576&t=44Y2p12QF0MmPAfj-0) don't have anything about updating comments in their social feed annotations.